homeassistant/components/kraken/config_flow.py

```diff
@@ -74,3 +53,32 @@ async def async_step_init(self, user_input=None):
         }
 
         return self.async_show_form(step_id="init", data_schema=vol.Schema(options))
+
+
+class KrakenConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for kraken."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
+
+    @staticmethod
+    @callback
+    def async_get_options_flow(
+        config_entry: config_entries.ConfigEntry,
+    ) -> KrakenOptionsFlowHandler:
+        """Get the options flow for this handler."""
+        return KrakenOptionsFlowHandler(config_entry)
+
+    async def async_step_user(
+        self, user_input: Optional[Dict[str, Any]] = None
```

Comment:
You need to use new style in type hints. There're only few places where they can't be used yet: `cast`, `isinstance`, type aliases. These places need to wait before Python 3.9 is the default, everywhere else we should use the new style.
```suggestion
        self, user_input: dict[str, Any] | None = None
```